Makefile

```diff
@@ -67,7 +64,7 @@ else ifeq ($(FFT),fftw3_f03)
   LIBFFT=-L$(FFTW3_PATH)/lib -lfftw3 -lfftw3f
 else ifeq ($(FFT),generic)
   INC=
-  LIBFFT=-lnvhpcwrapnvtx
+  LIBFFT=#-lnvhpcwrapnvtx
```

Comment:
I think this should be it's own target, e.g. FFT=nvhpc

Reply:
In one version of my Makefile I had the nvhpc target

Reply:
I have not tried to significantly improve the Makefile. The only objective was to have something working with nvidia, intel and gnu compilers. I had to comment this line because it is currently broken when using intel or gcc compiler. Feel free to further improve it or to push the other version of the Makefile you had ?